homeassistant/components/duckdns/__init__.py

```diff
@@ -40,15 +40,34 @@
 SERVICE_TXT_SCHEMA = vol.Schema({vol.Required(ATTR_TXT): vol.Any(None, cv.string)})
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+async def async_setup(hass, config):
+    """Import configuration from yaml."""
+    hass.data[DOMAIN] = {}
+    if DOMAIN not in config:
+        return True
+
+    conf = config.get(DOMAIN, {})
+
+    if not hass.config_entries.async_entries(DOMAIN):
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                DOMAIN, context={"source": SOURCE_IMPORT}, data=conf
+            )
+        )
+
```

Comment:
We need to raise an repair issue to inform the user that this can be removed from their YAML configuraiton.

Reply:
I've already raised a warning to the user for the removal of YAML config on the `config_flow.py` at the `async_step_import` method. Shall I move it to the `__init__.py` `async_setup` method?

Reply:
I think that makes sense (see also my comments below, where this returns :) )

Reply:
Moved it to `__init__.py`